### DISCUSSION TITLE
How do you right click on dashy tile on android ?

### DISCUSSION THREAD

**Discussion by hugalafutro** [Lissy93/dashy#508] [Q&A] [answered]
Holding on it just makes white outline around it. I prefer newtab over modal when on phone due to restricted screen space, so I was trying to "right click" on the dashy tile that defaults to modal in order to get the menu to show. Some googling suggested 2 finger tap as right click, either my fingers are too fat (they are) or it doesn't work. For now I edited the tiles and saved changes locally on phone.

On pc I am used to using middle click in chrome to "open in new tab". When I middle click on dashy tile I get a new tab with another dashy. Any way to get the underlying service in new tab regardless of what it defaults to?

**Comment by azerioxal** [comment, marked as answer]
I've submitted a PR that, pending approval, will fix this issue.

**Reply by lissy93** [reply]
Thanks @Ateroz - #541 is merged!
I've also got an item on my todo list to add a secondary opening method, which should be useful in your situation.

**Reply by hugalafutro** [reply]
Sorry I forgot a bit about this, but I just tried and it still doesn't seem to work on my android. I cleared local settings, closed the browser, loaded dashy in new tab and long pressing on dashy tile still only selects it with white border.

The whole reason is, I'd just love to have everything open in new tab on android and mostly in modals on pc. Just looking for a more convenient way than trying to maintain 2 sets of settings, so I was thinking having what I use on pc as "default" and when I happen to be on android I'd just right click to open in new tab.

**Reply by azerioxal** [reply]
I'm sorry, I must have misunderstood what you were referring to. Can you post a screenshot with what you're trying to open in a new tab indicated?

**Reply by lissy93** [reply]
@hugalafutro - Do you mean activating the context menu on long-press when on touch-enabled devices? Sorry, I misunderstood the original issue too.
If so, this should be possible, by making a custom directive (something [like this](https://blog.logrocket.com/building-a-long-press-directive-in-vue-3408d60fb511/)), so I will take a look and get it added :)

**Reply by hugalafutro** [reply]
> @hugalafutro - Do you mean activating the context menu on long-press when on touch-enabled devices? 

Hi that's exactly what I had in mind as I cannot find any way to show the right click menu on android. Sorry for being unclear, English is not my 1st language.

I made short video showing what happens when I long click (the zooming is where I tried to 2finger click to no avail):

https://user-images.githubusercontent.com/30209689/159156016-ca857ac2-8d9c-4571-bd3c-2d91b7a848fa.mp4

**Reply by lissy93** [reply]
Created ticket #586 to track this, fix coming in next PR :)
